Hazel/src/Hazel/Core.h

```diff
@@ -1,27 +1,37 @@
 #pragma once
 
-#ifdef HZ_PLATFORM_WINDOWS
+#if defined HZ_PLATFORM_WINDOWS
+        #define HZ_DEBUG_BREAK { __debugbreak(); }
```

Comment:
As @WhoseTheNerd [suggested](https://github.com/TheCherno/Hazel/pull/27/files/38151af3c6c33e93efc9d35881d19a482a42a5d1#r255294649):
> Wouldn't it be cleaner to use function macros?
> ```suggestion
>        #define HZ_DEBUG_BREAK()  __debugbreak()
>```

Also added these changes in the assert statements